sherpa/astro/tests/test_astro_data2.py

```diff
@@ -472,20 +472,22 @@ def test_416_a():
     mask = [False, False, False, True, True, True, False, False, False, False]
     assert pha.mask == pytest.approx(mask)
 
+    # With tabStops not set we use ~mask (as this is in channel space).
+    #
     pha.group_counts(3)
 
     # We have a simplified mask
-    mask = [True, True]
+    mask = [False] * 3 + [True] * 2 + [False] * 4
```

Comment:
This is exactly the mask defined in line 472. No need to put it in here again. Same is true for line 482/484 below.

Reply:
So, this is a hold over from when the mask was different. I'll clean up.

However I disagree with line 484 as that line is different (there's an extra bin).

Reply:
Actually, I mis-spole. This particular line *is* different, since the mask is now grouped. Hopefully the new version of the test will make it clearer.